### ISSUE TITLE
Running DeepLab on Cityscapes Semantic Segmentation Dataset

### ISSUE THREAD

**Issue by mattans** [tensorflow/models#7087]
I am running DeepLab on Cityscapes, using a pretrained model from the model zoo: http://download.tensorflow.org/models/deeplab_cityscapes_xception71_trainvalfine_2018_09_08.tar.gz

You did not include a mapping between class number ID and class textual name. Can you please attach such mapping?

Thank you.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by xiong224** [comment]
When you run this model on GPU,  whether you met the OOM ? Thanks a lot for your reply

**Comment by sdll** [comment]
@mattans, you can look up the label mapping [here](https://github.com/tensorflow/models/issues/6991). 

@xiong224, this is a known problem with a frozen graph which is [yet to be resolved](https://github.com/tensorflow/models/issues/4525). You might get better performance by loading the model from the checkpoint.

**Comment by fullprocess** [comment]
Does anyone know where to find the dimensions of the input tensor (is it 513x513x3?) for Deeplab?

**Comment by mattans** [comment]
> 
> 
> @mattans, you can look up the label mapping [here](https://github.com/tensorflow/models/issues/6991).
> 
> @xiong224, this is a known problem with a frozen graph which is [yet to be resolved](https://github.com/tensorflow/models/issues/4525). You might get better performance by loading the model from the checkpoint.

It should be 12 classes so it's not it. I'm using the semantic segmentation cityscapes dataset

**Comment by sdll** [comment]
The number of labels must be greater than 12, but there can also be more than that, like in the default PASCAL color map. The results of the tests I have done using this mapping seems to confirm that it is correct.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
